packages/rs-drive-abci/src/query/v0/mod.rs

```diff
@@ -783,13 +851,21 @@ impl<C> Platform<C> {
                             platform_version
                         ));
                     let serialized_identity = check_validation_result_with_data!(maybe_identity
-                        .map(|identity| identity.serialize_consume_to_bytes())
-                        .transpose());
+                        .ok_or_else(|| {
+                            QueryError::NotFound(format!(
+                                "identity {} not found",
+                                hex::encode(public_key_hash)
+                            ))
+                        })
+                        .and_then(|identity| identity
+                            .serialize_consume_to_bytes()
+                            .map_err(QueryError::Protocol)));
```

Comment:
Not sure that we need return basic errors as validation errors

Reply:
Will be fixed in the next PR